src/parsers/texttracks/srt/findEndOfCueBlock.ts

```diff
@@ -0,0 +1,56 @@
+
+/**
+ * Copyright 2015 CANAL+ Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Returns true if the given lines looks like the beginning of a cue.
+ * You should provide to this function only lines following "empty" lines.
+ * @param {string} line
+ * @returns {Boolean}
+ */
+function isStartOfCueBlock(lines : string[], index: number) : boolean {
```

Comment:
This is not really a isStart of cue block because for the following file (line added):
```
1 |   00:17:31.080 --> 00:17:32.200
2 |   Il y a un numéro.
3 |
4 |
5 |   00:17:33.120 --> 00:17:34.680
6 |   Ce numéro n'est pas en service.
```

Doing:
```js
isStartOfCueBlock(linified, 3); // NB: index 3 is line 4
```

Will return ``true`` whereas the cue actually begins at line 5 which means at index ``4``.